config/ember-try.js

```diff
@@ -107,6 +107,22 @@ module.exports = async function () {
         name: 'node-tests',
         command: 'yarn run nodetest',
       },
+      {
+        name: 'ember-concurrency-1.x',
+        npm: {
+          dependencies: {
+            'ember-concurrency': '^1.3.0',
```

Comment:
The two scenarios are failing for Bootstrap 3, probably due to this one missing (see other scenarios):

```suggestion
            'ember-concurrency': '^1.3.0',
            bootstrap: bootstrapVersion,
```